pooch/tests/test_core.py

```diff
@@ -499,6 +499,7 @@ def test_check_availability():
     assert not pup.is_available("not-a-real-data-file.txt")
 
 
+@pytest.mark.network
```

Comment:
I don't think we need to mark this test as `network`, since it's using the `ftpserver` fixture provided by [`pytest-localftpserver`](https://github.com/oz123/pytest-localftpserver), which runs a local ftp server for testing the `FTPDownloader`. Have you run in any problem trying to run this test without internet connection?
```suggestion
```

Reply:
We don't have `pytest-localftpserver` packaged for Fedora and it seems I included this solely going by the name of the test. It seems easy enough to package, though. I'll give that a shot and come back to you with a definitive answer.